### DISCUSSION TITLE
Система голосования за Лейтенанта полиции

### DISCUSSION THREAD

**Discussion by Kyper1** [octothorp-team/public-dobrograd#634] [Предложения]
- Чтобы стать лейтенантом будет необходимо уже быть полицейским, то есть обычным гражданским  и.т.д. будет запрещено  сразу стать лейтенантом.
- Голосование будет проводиться исключительно среди сотрудников полиции.

**Comment by Ggman77** [comment]
Да, это именно то, что нужно, ведь на сервере играет много краймеров, которые всегда голосуют против и чувак проигрывает голосование, я однозначно ЗА
